microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,105 @@ ms.date: 05/04/2021
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
+
+1. When DevTools has focus, press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). Or, you can click the (![Customize icon](../media/customize-devtools-icon-light-theme.png)) menu button in DevTools and then select **Open file**.  The Command Menu opens, displaying the **Open File** list:
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS).  The **Open File** dialog opens.
+   ![The Command Menu, displaying the Open File list](../media/resources-command-menu-empty.msft.png)
 
-   ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
+1. Start typing the filename and press `Enter` when the correct file is highlighted in the autocomplete box or select the file from the dropdown list.
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+   Type filename:
 
    ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Open resources in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+In the **Network** tool, you can inspect the resources that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
 
-<!-- need lead-in -->
+Show network resource details:
 
 ![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
 
-### Reveal resources in the Network tool from other panels
+#### Reveal resources in the Network tool from other panels
+
+- The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
 
-The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
+Pick resource to show in Network panel:
 
 ![Revealing resources in the Network panel.](../media/resources-sources-page-reveal-in-network-panel.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources in the Network panel
+## Browse resources in the Network tool
```

Comment:
```suggestion
## Browse and inspect resource files in the Network tool
```
by design, same as a new heading above.  Move the below content into the bottom of the first "Browse and inspect resource files in the Network tool" section above.